mmv1/third_party/terraform/services/monitoring/resource_monitoring_alert_policy_test.go

```diff
@@ -319,6 +319,8 @@ resource "google_monitoring_alert_policy" "full" {
     }
   }
 
+  severity     = "WARNING"
```

Comment:
Would it be possible to also include this in the basicCfg test, and demonstrate that the field can be changed eg. `ERROR` -> `WARNING`?